src/provider/model/ClientRegistry.ts

```diff
@@ -44,13 +45,17 @@ export class ClientRegistry {
      */
     public async tryLaunchApplication(appUuid: string): Promise<void> {
         const isRunning = await this._environment.isApplicationRunning(appUuid);
-
         if (!isRunning) {
             const storedApplication = this._store.state.applications.get(appUuid);
             if (storedApplication) {
-                await this._environment.startApplication(storedApplication);
+                try {
+                    await this._environment.startApplication(storedApplication);
+                } catch (error) {
+                    throw new LaunchApplicationError('Could not launch application', error);
+                }
             } else {
-                console.warn('Could not find application initialization data for the application with uuid ' + appUuid + ' in the database.');
+                // eslint-disable-next-line max-len
```

Comment:
The line below doesn't actually exceed the max length :-)

Reply:
Removed